src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2549,13 +2549,14 @@ because xbuild doesn't support framework reference assemblies.
 		ToolExe="$(ZipalignToolExe)"
 	/>
 	<AndroidApkSigner Condition=" '$(AndroidUseApkSigner)' == 'true' "
+		ApkSignerJar="$(ApkSignerJar)"
 		ApkToSign="$(ApkFileSigned)"
 		KeyStore="$(_ApkKeyStore)"
 		KeyAlias="$(_ApkKeyAlias)"
 		KeyPass="$(_ApkKeyPass)"
 		StorePass="$(_ApkStorePass)"
-		ToolPath="$(AndroidSdkBuildToolsBinPath)"
-		ToolExe="$(ApkSignerToolExe)"
+		ToolPath="$(JavacToolPath)"
```

Comment:
I believe that this should be `$(JavaToolPath)`, and the following line `$(JavaToolExe)`, *not* *`Javac`*.